src/sha3/rtl/sha3_reg.rdl

```diff
@@ -420,7 +420,6 @@ addrmap sha3_reg {
             For the interrupt to be raised, the message FIFO must also have been full previously.
             Otherwise, the hardware empties the FIFO faster than software can fill it 
             and there is no point in interrupting the software to inform it about the message FIFO being empty.";
```

Comment:
Do we need a note somewhere that explains the W1C behaviour? (And is the RDL file the right place for it? I genuinely don't know)

Reply:
I think the RDL file is probably the right way of doing this. This behaviour may need to change once we replace the OpenTitan-style register file with the RDL-style register file, so I think I'll leave this for now to clear up in a future version if that's OK.